builtin/providers/digitalocean/resource_digitalocean_droplet_test.go

```diff
@@ -144,6 +144,56 @@ func TestAccDigitalOceanDroplet_ResizeWithOutDisk(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanDroplet_ResizeOnlyDisk(t *testing.T) {
```

Comment:
It's possible to merge this test into `TestAccDigitalOceanDroplet_ResizeWithOutDisk` as it's the strict superset of the latter (and name it just `TestAccDigitalOceanDroplet_Resize`).